subsys/logging/log_backend_uart.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <logging/log_backend_uart.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <device.h>
+#include <uart.h>
+
+int char_out(u8_t *data, size_t length, void *ctx)
+{
+	struct device *dev = (struct device *)ctx;
+
+	for (size_t i = 0; i < length; i++) {
+		uart_poll_out(dev, data[i]);
+	}
```

Comment:
minor coding style issue: empty line after `}`  (unless the next line is also a `}`